packages/calling-stateful-client/review/beta/calling-stateful-client.api.md

```diff
@@ -251,6 +253,13 @@ export interface OptimalVideoCountFeatureState {
     maxRemoteVideoStreams: number;
 }
 
+// @public
+export interface RaiseHandCallFeature {
+    raisedHands: RaisedHand[];
+    // (undocumented)
+    userRaisedHand?: RaisedHand;
```

Comment:
I dont like this name. can we change this to "myRaisedHand" or something? or localRaisedHand? Can we bring this up in the feature weekly?